server/utils/match/index.ts

```diff
@@ -195,6 +190,37 @@ export const occupancyViewSummaryListForMatchingDetails = (
   ]
 }
 
+export const textRow = (label: string, value: string): SummaryListItem => ({
+  key: { text: label },
+  value: { text: value },
+})
+
+export const htmlRow = (label: string, value: string): SummaryListItem => ({
+  key: { text: label },
+  value: { html: value },
+})
+
+export const spaceBookingConfirmationSummaryListRows = (
+  placementRequest: PlacementRequestDetail,
+  premises: Cas1PremisesSummary,
+  arrivalDate: string,
+  departureDate: string,
+  criteria: Array<Cas1SpaceBookingCharacteristic>,
+): Array<SummaryListItem> => {
+  return [
+    textRow('Approved Premises', premises.name),
+    // textRow('Address', premises.fullAddress),
```

Comment:
could remove commented line

Reply:
Address is now available from API and showing, so this is uncommented